docs/ref/contrib/postgres/fields.txt

```diff
@@ -920,3 +920,24 @@ define :class:`~django.contrib.postgres.constraints.ExclusionConstraint`. See
 docs/current/rangetypes.html#RANGETYPES-INCLUSIVITY>`_.
 
 .. _different bounds: https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-IO
+
+``Serial`` fields
+=================
+
+.. versionadded:: 5.2
+
+.. class:: SerialField(**options)
+
+    An automatically incrementing :class:`~django.db.models.IntegerField`,
+    much like :class:`~django.db.models.AutoField` except that it does not have
+    to be a primary key, and is not limited to a single instance per model.
```

Comment:
Maybe we should mention that it's backed by the Postgres `serial` type?

Reply:
ok